pages/knowledge/colours-and-contrast/user-interface-components/README.md

```diff
@@ -0,0 +1,52 @@
+---
+navigation_title: "Colour contrast for UI components"
+position: 4
+changed: "2019-04-21"
+---
+
+# Colour contrast for User Interface components
+
+**“User Interface (UI) components” are interactive elements, ranging from simple buttons and input fields to complex UI widgets. Just as for text there is a general minimal contrast level that needs to be reached to make their functionality and content accessible.**
+
+## Minimal contrast ratio
+
+Version 2.1 of the [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast) of `3:1` against adjacent colour(s) for all User Interface (UI) components.
+
+## Example: form controls
+
+The following screenshot shows a very simple contact form
+
+![Low contrast form](_media/low-contrast-form.png)
+
+The borders of the input fields and the background colour of the “Send message” button have an insufficient contrast ratio of `1.5:1`. 
+
+Because of that, certain users may have a hard time finding the input fields and enter information. 
+
+They might also struggle to identify the button by its typical shape. However, the button label adds a lot of valuable information, making this case less urgent or even negligible: it has sufficient contrast, it offers a clear invitation to interact by using a verb (which is better than a generic “OK”) and its visual appearance (font, size) is different from the other text elements surrounding it.
+
+## Example: tablists
+
+Tablists convey visually that there is a list of items, and that the content for the currently active item is displayed below. The button-like appearance of the tabs indicates that a user can select them to display their associated content.
```

Comment:
I'd love to add a hint about the fact that only one item can be shown at a time. Original text (can be improved):

> The button-like style of the items indicates that a user can click on them, making the clicked item the active one and displaying its content.

Reply:
**To Do:** Add info that there’s only one item selectable at a time (similar to radio buttons)

Reply:
New text:

> Tablists convey visually that there is a list of items, and that the content for the currently active item is displayed below. The button-like appearance of the tabs indicates that a user can select them to display their associated content. The overall visual structure where the active tab is "in front" of the inactive tabs indicates that only one tab can be active at a time (unlike other visual patterns such as tags or labels).